src/localize/languages/pt-PT.json

```
@@ -0,0 +1,470 @@
+        "alignments": {
+          "matching": "Mesmo alinhamento do menu",
+          "opposing": "Opor-se ao alinhamento do menu"
+        },
+        "camera_ui": "Camera",
+        "cameras": "Selecionar câmera",
+        "clips": "Clipes",
+        "download": "Baixe a mídia do evento",
+        "enabled": "Botão ativado",
+        "expand": "Expandir",
+        "frigate": "Frigate menu / Visualização padrão",
+        "fullscreen": "Tela cheia",
+        "icon": "Ícone",
+        "image": "Imagem",
+        "live": "Ao vivo",
+        "media_player": "Enviar para o reprodutor de mídia",
+        "priority": "Prioridade",
+        "snapshots": "Instantâneos",
+        "substreams": "substreams",
+        "timeline": "Linha do tempo"
+      },
+      "position": "Posição do menu",
+      "positions": {
+        "bottom": "Posicionado na parte inferior",
+        "left": "Posicionado à esquerda",
+        "right": "Posicionado à direita",
+        "top": "Posicionado no topo"
+      },
+      "style": "Estilo do menu",
+      "styles": {
+        "hidden": "Menu oculto",
+        "hover": "Menu suspenso",
+        "none": "Sem menu",
+        "outside": "Menu externo",
+        "overlay": "Menu sobreposto"
+      }
+    },
+    "overrides": {
+      "info": "Esta configuração do cartão especificou manualmente as substituições configuradas que podem substituir os valores mostrados no editor visual, consulte o editor de código para visualizar/modificar essas substituições"
+    },
+    "performance": {
+      "features": {
+        "animated_progress_indicator": "Animação na barra de progresso",
+        "editor_label": "Editor de rotulos",
+        "media_chunk_size": "Tamanho do ficheiro"
+      },
+      "profile": "Perfil",
+      "profiles": {
+        "high": "Alto",
+        "low": "Baixo"
+      },
+      "style": {
+        "border_radius": "Tamanho da borda",
+        "box_shadow": "Caixa de Fundo",
+        "editor_label": "Editor de rotulos"
+      },
+      "warning": "Avisos"
+    },
+    "view": {
+      "camera_select": "Visualização de câmeras recém-selecionadas",
+      "dark_mode": "Modo escuro",
+      "dark_modes": {
+        "auto": "Automático",
+        "off": "Desligado",
+        "on": "Ligado"
+      },
+      "default": "Visualização padrão",
+      "scan": {
+        "enabled": "Modo scan ativado",
+        "scan_mode": "Modo scan",
+        "show_trigger_status": "Pulsar borda quando acionado",
+        "untrigger_reset": "Redefinir a visualização para o padrão após desacionar",
+        "untrigger_seconds": "Segundos após a mudar para o estado inativo para desacionar"
+      },
+      "timeout_seconds": "Redefinir para a visualização padrão X segundos após a ação do usuário (0 = nunca)",
+      "update_cycle_camera": "Percorrer as câmeras quando a visualização padrão for atualizada",
+      "update_force": "Forçar atualizações do cartão (ignore a interação do usuário)",
```

Brazilian expression not used in pt_PT
```suggestion
      "update_force": "Forçar atualizações do cartão (ignore a interação do Utilizador)",
```